Translate docs/hello-world.md (#10)
reactjs/nl.reactjs.org: content/docs/hello-world.md

next: introducing-jsx.html
---

> Het meest eenvoudige React-voorbeeld ziet er zo uit:

Author: @Keraito Dutch has awful rules concerning "samenstellingen" with foreign words, which is essentially what we're trying to form here. Personally I think the hyphen is needed to be 100% grammatically correct, but it might be better for readability to mitigate it.

);
```

> Het weergeeft een kop met de tekst "Hello, world!" op de pagina.

Reviewer: ```suggestion
Het toont een kop met de tekst "Hello, world!" op de pagina.
```

An alternative with "weergeven" could be 
```Het geeft een kop met de tekst "Hello, world!" weer op de pagina.```

[Probeer het op CodePen](codepen://hello-world)

Klik op de bovenstaande link om een online editor te openen.
Voel je vrij om wat aanpassingen te maken en te zien hoe deze de uitvoer beïnvloeden.
De meeste pagina's in deze gids hebben dit soort bewerkbare voorbeelden.

## Hoe Lees Je Deze Gids {#how-to-read-this-guide}

In deze gids zullen we de bouwstenen van React-applicaties onderzoeken: elementen en componenten.

> Van zodra je ze onder de knie hebt, kan je complexe applicaties ontwikkelen uit kleine, herbruikbare stukjes.

Reviewer: ```suggestion
Als je deze onder de knie hebt, kan je complexe applicaties ontwikkelen uit kleine, herbruikbare stukjes.
```

"Van zodra" is not something a Dutchman would use as an expression. It would either be "zodra", or more likely the simpler "als".

Author: I agree with both of you here. I will opt for "wanneer".

>Tip
>

> >Deze gids is bedoeld voor zij die liever **concepten stap voor stap leren**. Als je liever al doende leert, bekijk dan onze [praktische tutorial](/tutorial/tutorial.html). Mogelijk vind je dan dat deze gids en de tutorial elkaar aanvullen.

Reviewer: ```suggestion
>Deze gids is bedoeld voor mensen die liever **concepten stap voor stap leren**. Als je liever al doende leert, bekijk dan onze [praktische tutorial](/tutorial/tutorial.html). Je zult dan zien dat deze gids en de tutorial elkaar aanvullen.
```

Author: @Keraito I would opt for "je" for this kind of a text, definitely considering the potentially younger audience.

Reviewer: Agreed, @jochenboesmans. Using "je" seems to fit better with our target audience.

Dit is het eerste hoofdstuk in een stap-voor-stapgids over de belangrijkste React-concepten.

> Je kan een lijst van alle hoofdstukken vinden in de navigatiezijbalk.

Reviewer 2: ```suggestion
Een lijst van alle hoofdstukken is te vinden in de navigatie zijbalk.
```

> Als je dit leest vanaf een mobiel apparaat, kan je de navigatie openen door op de knop in de rechterbenedenhoek van je scherm te drukken.

Reviewer 2: This one is essential:
```suggestion
Als je dit leest vanaf een mobiel apparaat, dan kun je de navigatie openen door op de knop in de rechterbenedenhoek van je scherm te drukken.
```

These ones I will leave up to you:
```suggestion
Indien je dit leest vanaf een mobiel apparaat, kan je de navigatie openen door op de knop in de rechterbenedenhoek van je scherm te drukken.
```

```suggestion
Als je dit leest vanaf een mobiel apparaat, kan je bij de navigatie komen via de knop in de rechterbenedenhoek van je scherm.
```

> Elk hoofdstuk in deze gids bouwt verder op de kennis die in voorgaande hoofdstukken werd opgebouwd.

Reviewer 2: Repetitive usage of "opbouwen" doesn't read too nice imo
```suggestion
Elk hoofdstuk in deze gids bouwt verder op de kennis uit voorgaande hoofdstukken.
```

> **Het grootste deel van React kan je leren door de hoofdstukken “Algemene Concepten” te lezen in de volgorde waarin ze verschijnen in de zijbalk**.

Reviewer 2: ```suggestion
**Het grootste deel van React kan je leren door de “Algemene Concepten” hoofdstukken te lezen in de volgorde waarin ze verschijnen in de zijbalk**.
```

Author: I think there'd technically be a hyphen due again with this change. Waiting for the result of that discussion.

[“JSX: Een Introductie”](/docs/introducing-jsx.html) is bijvoorbeeld het volgende hoofdstuk.

> ## Veronderstellingen Op Vlak Van Kennis {#knowledge-level-assumptions}

Reviewer: ```suggestion
## Aannames over je kennisniveau {#knowledge-level-assumptions}
```

> React is een JavaScript library en dus zullen we veronderstellen dat je een basisbegrip hebt van de JavaScript taal.

Reviewer: ```suggestion
React is een JavaScript library en dus zullen we aannemen dat je een basisbegrip hebt van de JavaScript taal.
```
There's no difference in meaning between "aannemen" and "veronderstellen", but the former is a bit easier to understand, I think.

> **Als je hier niet erg zeker over bent, raden we [deze JavaScript tutorial](https://developer.mozilla.org/nl/docs/Web/JavaScript/A_re-introduction_to_JavaScript) aan om je niveau te testen** en om je in staat te stellen deze gids mee te volgen zonder te verdwalen.

Reviewer 2: I think it's better to not directly reference the tutorial we're linking, just like the English version, as it causes an assumption that it's an official tutorial or that going through the tutorial will guarantee readers not having problems with React.
```suggestion
**Als je hier niet erg zeker over bent, raden we [een JavaScript tutorial](https://developer.mozilla.org/nl/docs/Web/JavaScript/A_re-introduction_to_JavaScript) aan om je niveau te testen** en om je in staat te stellen deze gids mee te volgen zonder te verdwalen.
```
```suggestion
**Als je hier niet erg zeker over bent, raden we [deze JavaScript tutorial](https://developer.mozilla.org/nl/docs/Web/JavaScript/A_re-introduction_to_JavaScript) aan om je niveau te testen** en om je in staat te stellen deze gids te volgen zonder te verdwalen.
```

> Het zou je tussen 30 minuten en een uur kunnen kosten, maar hierdoor zal je React en JavaScript in ieder geval niet tegelijkertijd moeten leren.

Reviewer: ```suggestion
Misschien ben je er 30 minuten tot een uur mee bezig, maar daardoor hoef je in ieder geval niet tegelijkertijd React en JavaScript leren.
```

>Opmerking
>
>Deze gids gebruikt af en toe wat van de nieuwere JavaScript syntax in de voorbeelden.
Als je in de laatste paar jaar niet met JavaScript hebt gewerkt, zouden [deze drie aandachtspunten (Engels)](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) je op weg moeten helpen.

> ## Laten We Eraan Beginnen! {#lets-get-started}

Reviewer 2: "Eraan" can be omitted, but leaving that up to your pref

> Als je wat naar beneden scrolt, zal je een link vinden naar het [volgende hoofdstuk van deze gids](/docs/introducing-jsx.html) juist boven de voettekst van de website.

Reviewer: ```suggestion
Als je wat naar beneden scrolt, zal je een link vinden naar het [volgende hoofdstuk van deze gids](/docs/introducing-jsx.html) net boven de voettekst van de website.
```
